source/extensions/all_extensions.bzl

```diff
@@ -42,8 +42,11 @@ def envoy_all_http_filters():
 # All network-layer filters are extensions with names that have the following prefix.
 _network_filter_prefix = "envoy.filters.network"
 
+# All thrift filters are extensions with names that have the following prefix.
+_thrift_filter_prefix = "envoy.filters.thrift"
```

Comment:
> Is this because they're registered under a different naming convention? (instead of envoy.filters.network) Will this only pick up a network filter? May want to clarify in comment.

Thrift_proxy filter has it own filters(sub-filters?). There are two sub-filters there.  If we only load `envoy.filters.network.*` and `thrift_router` or `thrift_ratelimit` filter exists in the config of thrift_proxy, an error will occur because cannot find it in the factory. It's a **different** factory from network-filter but it is necessary for thrift_proxy to run. (source/extensions/filters/network/thrift_proxy/filters/well_known_names.h)

If these two filters are only designed for thrift, I think their name should begin with `envoy.filters.network.thrift` but now they don't have `network` prefix. So I manually added them here.